src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 0;
+
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days < 7) {
+    cost = (days * 40) - 20;
+
+    return cost;
+  }
```

Comment:
what is 40? what is 20? what is 50? i have no idea
also do not create a variable just to then return it

Reply:
Thanks have made changes